main_test.go

```diff
@@ -875,6 +876,19 @@ sliceptr.StrVector{"1", "2", "3", "4"}
 	})
 }
 
+func TestUnicode(t *testing.T) {
+	t.Parallel()
+	path := "_examples/unicode"
+	testPkg(t, pkg{
+		path: path,
+		lang: []string{"py2", "py2-cffi", "py3-cffi", "pypy2-cffi", "pypy3-cffi"},
+		want: []byte(`encoding.HandleString(bytestr) -> Python byte string
+encoding.HandleString(unicodestr) -> Python Unicode string 🐱
```

Comment:
Please remove emoji

Reply:
I assume the emoji is there to test utf-8 :)

Reply:
Ah.. you are right.